src/PhpSpec/Console/ContainerAssembler.php

```diff
@@ -389,7 +389,7 @@ private function setupLocator(IndexedServiceContainer $container)
 
                 $config = array_merge($defaults, $suite);
 
-                if (!is_dir($config['src_path'])) {
+                if (!is_dir($config['src_path']) && !empty($config['src_path'])) {
```

Comment:
Yes you are correct I made that small change as it makes more sense checking if it is not empty first.